### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by JensTrier**
Updates to:

- TeamsCallingPolicy
- TeamsEmergencyCallingPolicy
- CsUser
- CsOnlineLisLocation
- CsOnlineLisCivicAddress
- CsPhoneNumberAssignment

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -94,13 +94,12 @@ Accept wildcard characters: False
 ```
 
 ### -AllowCallRedirect
-Setting this parameter provides the ability to configure call redirection capabilities on Teams Phones. 
+Setting this parameter is enabling local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallingPolicy.md:97]:
```suggestion
Setting this parameter enables local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.

```

File: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md [hunk 2]
```diff
@@ -94,13 +94,12 @@ Accept wildcard characters: False
...
-Setting this parameter provides the ability to configure call redirection capabilities on Teams Phones. 
+Setting this parameter is enabling local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.
 
 Valid options are:
-- Enabled: Users will have the ability to redirect received calls.
-- Disabled: Users will not have such an ability.
-- UserOverride: This option is not available for use. There is no user interface implemented for its management.
-
+- Enabled: Enable the user to redirect an incoming call.
+- Disabled: User will not have such ability.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallingPolicy.md:101]:
```suggestion
- Enabled: Enables the user to redirect an incoming call.
- Disabled: The user cannot redirect an incoming call.

```

File: skype/skype-ps/skype/Set-CsOnlineLisCivicAddress.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ ms.reviewer:
 Use the \`Set-CsOnlineLisCivicAddress\` cmdlet to modify an existing civic address which has not been validated.
 Validated civic addresses cannot be modified.
 
+**Note:** This cmdlet is only available for Microsoft Internal use.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineLisCivicAddress.md:19]:
```suggestion
**Note:** This parameter is for Microsoft internal use only.
```

File: skype/skype-ps/skype/Set-CsOnlineLisLocation.md [hunk 4]
```diff
@@ -44,19 +44,14 @@ Set-CsOnlineLisLocation -LocationId 5aa884e8-d548-4b8e-a289-52bfd5265a6e -Locati
...
-
-This example changes the Elin of the location specified by its location identity.
-
 ## PARAMETERS
 
 ### -CivicAddressId
 Specifies the unique identifier of the civic address that contains the location to be modified.
 Civic address identities can be discovered by using the \`Get-CsOnlineLisCivicAddress\` cmdlet.
 
+**Note:** This parameter is for Microsoft Internal use only.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineLisLocation.md:53]:
```suggestion
**Note:** This parameter is for Microsoft internal use only.
```

File: skype/skype-ps/skype/Set-CsOnlineLisLocation.md [hunk 5]
```diff
@@ -202,8 +208,9 @@ Accept wildcard characters: False
 ```
 
 ### -Elin
-Specifies the Emergency Location Identification Number.
-This is used in Direct Routing EGW scenarios.
+Specifies the Emergency Location Identification Number. This is used in Direct Routing EGW scenarios.
+
+**Note:** You can set or change the ELIN, but you can't clear its value. If you need to clear the value, you should re-create the location.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineLisLocation.md:213]:
```suggestion
**Note:** You can set or change the ELIN, but you can't clear its value. If you need to clear the value, you should recreate the location.
```

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 6]
```diff
@@ -103,12 +103,12 @@ Accept wildcard characters: False
 ```
 
 ### -AllowCallRedirect
-Setting this parameter provides the ability to configure call redirection capabilities on Teams Phones. 
+Setting this parameter is enabling local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:106]:
```suggestion
Setting this parameter enables local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.
```

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 7]
```diff
@@ -103,12 +103,12 @@ Accept wildcard characters: False
...
 
 Valid options are:
-- Enabled: Users will have the ability to redirect received calls.
-- Disabled: Users will not have such an ability.
-- UserOverride: This option is not available for use. There's no user interface implemented for its management.
+- Enabled: Enable the user to redirect an incoming call.
+- Disabled: User will not have such ability.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:110]:
```suggestion
- Enabled: Enables the user to redirect an incoming call.
- Disabled: The user cannot redirect an incoming call.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 8]
```diff
@@ -202,6 +202,8 @@ be flagged as a duplicate number. This is due to the fact that, depending on you
 routes your call to an Exchange Auto Attendant. Conversely, dialing just the extension (51297) or using Skype for Business to dial the number 1-425-555-1297 extension 51297 will route your call
 directly to the user.
 
+For Direct Routing phone numbers in GCC High and DoD cloud instances it is not supported to assign a base phone number to a user or resource account, if you already have other users or resource accounts assigned with phone numbers with the same base phone number + extensions or vice versa. For instance if you have a user with the assigned phone number +14255551200;ext=123 you can't assign the phone number +14255551200 to another user or resource account or if you have a user or resource account with the assigned phone number +14255551200 you can't assign the phone number +14255551200;ext=123 to another user or resource account. It is supported to assign phone numbers with same base number but different extensions to users and resource accounts. For instance you can have a user with +14255551200;ext=123 and another user with +14255551200;ext=124.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:205]:
```suggestion
For Direct Routing phone numbers in GCC High and DoD cloud instances, assigning a base phone number to a user or resource account is not supported if you already have other users or resource accounts assigned phone numbers with the same base phone number and extensions or vice versa. For instance, if you have a user with the assigned phone number +14255551200;ext=123 you can't assign the phone number +14255551200 to another user or resource account or if you have a user or resource account with the assigned phone number +14255551200 you can't assign the phone number +14255551200;ext=123 to another user or resource account. Assigning phone numbers with the same base number but different extensions to users and resource accounts is supported. For instance, you can have a user with +14255551200;ext=123 and another user with +14255551200;ext=124.
```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 9]
```diff
@@ -27,6 +27,8 @@ This cmdlet displays information about one or more phone numbers. You can filter
 
 Returned results are sorted by TelephoneNumber in ascending order.
 
+If you are using both -Skip X and -Top Y for filtering, the returned results will first be skipped by X and then the top Y results will be returned.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:30]:
```suggestion
If you are using both -Skip X and -Top Y for filtering, the returned results will first be skipped by X, and then the top Y results will be returned.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/Set-CsOnlineLisCivicAddress.md (hunk 3 above)
Comment (JensTrier) [reply skype/skype-ps/skype/Set-CsOnlineLisCivicAddress.md:19]: @JohanFreelancer9 it is the full cmdlet that is only available to Microsoft

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks @JohanFreelancer9! There is one suggested change I disagree with, since it is the cmdlet itself that is only available for Microsoft

**Comment by JohanFreelancer9** [comment]
> > Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.
> 
> Thanks @JohanFreelancer9! There is one suggested change I disagree with, since it is the cmdlet itself that is only available for Microsoft

Thank you for pointing that out to me @JensTrier! Please just commit the minor suggestion to correct the capitalization. Thanks again.

**Comment by shivtorov** [comment]
Closes #10933

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**